src/main/resources/assets/sodium/lang/ru_ru.json

```
@@ -0,0 +1,66 @@
+  "sodium.options.entity_distance.tooltip": "Управляет дальностью видимости сущностей. Высокие значения увеличивают расстояние отображения за счёт частоты кадров.",
+  "sodium.options.entity_shadows.tooltip": "Если включено, простые тени будут отображаться под мобами и другими сущностями.",
+  "sodium.options.vignette.name": "Виньетирование",
+  "sodium.options.vignette.tooltip": "Если включено, на экране игрока будет отображаться эффект виньетирования. Вряд ли повлияет на частоту кадров, если ваш ГП не ограничен скоростью заполнения пикселей.",
+  "sodium.options.mipmap_levels.tooltip": "Управляет количеством уровней детализации для текстур блоков. Более высокие значения обеспечивают лучший вид на расстоянии, но могут негативно повлиять на производительность с большим количеством анимированных текстур.",
+  "sodium.options.use_block_face_culling.name": "Отбраковка поверхностей блоков",
+  "sodium.options.use_block_face_culling.tooltip": "Если включено, на рендеринг будут выводиться только видимые для камеры поверхности блоков. Это может исключить большое количество поверхностей на ранней стадии отрисовки, экономя пропускную способность памяти и время для ГП. Некоторые пакеты ресурсов могут иметь проблемы с этой опцией, поэтому попробуйте отключить её, если вы видите дыры в блоках.",
+  "sodium.options.use_fog_occlusion.name": "Окклюзия тумана",
+  "sodium.options.use_fog_occlusion.tooltip": "Если включено, чанки, определённые как полностью скрытые в тумане, не будут отображаться ради улучшения производительности. Улучшение может быть более значительным, когда эффект тумана сильнее (например, под водой), но в некоторых сценах это может привести к нежелательным визуальным артефактам между небом и туманом.",
+  "sodium.options.use_entity_culling.name": "Отбраковка сущностей",
+  "sodium.options.use_entity_culling.tooltip": "Если включено, сущности, обнаруженные вне видимых чанков, не будут выводиться на рендеринг. Это может помочь улучшить производительность, исключив рендеринг сущностей, находящихся под землёй или за стенами.",
+  "sodium.options.use_particle_culling.name": "Фильтрация частиц",
+  "sodium.options.use_particle_culling.tooltip": "Если включено, будут отображаться только те частицы, которые определены как видимые. Это может обеспечить значительное улучшение частоты кадров, когда рядом находится много частиц.",
+  "sodium.options.animate_only_visible_textures.name": "Анимировать только видимое",
+  "sodium.options.animate_only_visible_textures.tooltip": "Если включено, будут обновляться лишь видимые анимированные текстуры. Это может значительно повысить частоту кадров на некоторых устройствах, особенно с \"тяжёлыми\" пакетами ресурсов. Если у каких-то текстур возникают проблемы с анимацией, попробуйте отключить эту опцию.",
+  "sodium.options.cpu_render_ahead_limit.name": "Лимит пре-рендеринга",
```

I guess, this is the best translation. Also, an extended version of this phrase is used in the Nvidia Driver Control Panel
```suggestion
  "sodium.options.cpu_render_ahead_limit.name": "Лимит подготовки кадров",
```